app/models/manageiq/providers/ibm_cloud/cloud_object_storage/storage_manager/event_catcher.rb

```diff
@@ -0,0 +1,3 @@
+class ManageIQ::Providers::IbmCloud::CloudObjectStorage::StorageManager::EventCatcher < ManageIQ::Providers::BaseManager::EventCatcher
```

Comment:
Unless we're going to implement an event catcher in this PR lets delete these scaffolding classes